Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM node:16.17.0-alpine
+WORKDIR '/app'
+
+COPY package.json .
+RUN npm install
+COPY . .
+
+CMD ["npm","run","dev"]
+
+
+
+# if facing error while building image, paste there two command in terminal.
+
+# export DOCKER_BUILDKIT=0
+#export COMPOSE_DOCKER_CLI_BUILD=0
```

Comment:
```suggestion
#export COMPOSE_DOCKER_CLI_BUILD=0

```

Reply:
they are commented out, i dont think they are causing it

Reply:
not the comment - but the missing endline

Reply:
it is running fine, i tried running it.

Reply:
I am talking for code style/codacy

Reply:
Ohh

Reply:
What should we do now?